client/components/chart/index.js

```diff
@@ -219,19 +219,25 @@ class Chart extends Component {
 					<span className="woocommerce-chart__title">{ title }</span>
 					{ width > WIDE_BREAKPOINT && legendDirection === 'row' && legend }
 					{ this.renderIntervalSelector() }
-					<div className="woocommerce-chart__types">
+					<div className="woocommerce-chart__types" role="radiogroup">
 						<IconButton
 							className={ classNames( 'woocommerce-chart__type-button', {
 								'woocommerce-chart__type-button-selected': type === 'line',
 							} ) }
 							icon={ <Gridicon icon="line-graph" /> }
+							title={ __( 'Line chart', 'wc-admin' ) }
+							aria-checked={ type === 'line' }
+							role="radio"
```

Comment:
Since navigating between menu items is handled by the arrow keys, we can also disable the tab stops on the unselected buttons, for example on this button: `tabIndex={ type === 'line' ? 0 : -1 }`